frappe/core/doctype/transactionlog/transactionlog.js

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2017, Frappe Technologies and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('TransactionLog', {
```

Comment:
Doctype name should be with spaces `Transaction Log`, we never used without space.
https://github.com/frappe/erpnext/wiki/Naming-Guidelines#doctype-naming